[ACR-1155] remove deprecated enum (#1059)
Samsung/tizen-docs: docs/application/native/guides/multimedia/media-playback.md

The supported audio formats include MP3, AAC, WMA, M4A, 3GA, WAV, and AMR. The available formats depend on the target device.

> S/W and H/W audio codec could be selected by player API and the default codec type should be checked by calling player_get_audio_codec_type().

Reviewer: Suggest rephrasing, please verify:

```suggestion
Software and hardware code can be selected using the player API. You must verify the codec by calling `player_get_audio_codec_type()`.
```

Reviewer 2: Hello, @surbhig12 
The "S/W and H/W audio codec" is changed to "Software and hardware code", and I hope you to check the update again. With this update, isn't the "audio codec" become "code"?

Author: If the 'codec' is not normal, we can replace it to 'decoder'.

Reviewer: @eunhaechoi,

In my opinion, in this scenario decoder may not be right. 
codec means: A device or program that compresses data to enable faster transmission and decompresses received data. 
decoder means: a device capable of converting audio or video signals into a different form, for example from digital to analogue.
In this scenario, I think we are referring to codec

Please correct me if I am wrong

```suggestion
Software and hardware audio codec can be selected using the player API. You must verify the codec by calling `player_get_audio_codec_type()`.
```

Reviewer 2: Additional question to @surbhig12 

**Original :** the default codec type should be checked by calling player_get_audio_codec_type().
**Suggested :** You must verify the codec by calling `player_get_audio_codec_type()`.

According to the API Reference, `player_get_audio_codec_type()` is called to get the default codec type information of the audio decoder. As the default codec type can be different depending on the device, I think the original sentence is focusing on "To check the information what the default codec type is" as the sentence itself.
However, the suggested sentence is focusing on more general meaning ("verify codec"), so to me, it seams the meaning get blurred a little bit. Would you please check again the sentence?

If I understood something wrong, please let me know.

**Reference :** http://10.113.164.110/native/5.5/mobile/group__CAPI__MEDIA__PLAYER__MODULE.html#ga5b88c6383ed311fdd6ef2e7530505d17
http://10.113.164.110/native/5.5/mobile/group__CAPI__MEDIA__PLAYER__MODULE.html#gacb9bebbd139f12bb45ad2a8ddd332da4

Reviewer: @mijin85cho, thank you for the detailed explanation.
Would the following be okay:

Software and hardware audio codec can be selected using the player API. You must verify the codec type by calling `player_get_audio_codec_type()`.

Reviewer 2: @surbhig12 
- Well... I think "verify" is too heavy to the meaning I understood because it is _"to get the default codec type information / It can also fail to get the default codec type because of wrong parameters or something else"_, not _"to confirm the codec type information."_
As the "verify" sounds like "confirming what kind of codec type information and whether it is valid/or not" to me, if possible I hope to use "check" rather than "verify".
Anyway if you insist, we can use "verify" also.

- I thought the "default codec type" was the point, so it would be better to add back the "default" to your suggestion.

Reviewer: @mijin85cho, sorry for the miss and thank you for correcting me:

Please find the updated sentence:

Reviewer 2: @surbhig12 Thanks for considering my opinion. I think the latest sentence is good.

> Software and hardware audio codec can be selected using the player API. You must check the default codec type by calling 'player_get_audio_codec_type()'.

Reviewer 2: Instead of ', please use \` to use the code tag.

`player_get_audio_codec_type()`

Author: I pushed new commit. thank you.

> You can use multiple instances of the player to play several audio data streams concurrently if S/W codec is selected. This means that your application can [play multiple uncompressed audio files](#play) at the same time.

Reviewer: Suggest rephrasing, please verify:

```suggestion
If the software codec is selected, you can use multiple instances of the player to play several audio data streams concurrently. This implies that your application can [play multiple uncompressed audio files](#play) at the same time.
```

The following figure illustrates the general player state changes.

**Figure: Player state changes**

![Player state changes](./media/using_players_state_changes.png)

> The `player_interrupted_cb()` callback could be called due to the resource restriction in the system. The player state will be one of IDLE, READY or PAUSED according to the resource management policy. You must check the state of the current instance and take appropriate action.

Reviewer: Suggest rephrasing, please verify:

```suggestion
The `player_interrupted_cb()` callback could be called due to the resource restriction in the system. The player state could be IDLE, READY, or PAUSED based on the resource management policy. You must check the state of the current instance and take appropriate action.
```

Reviewer 2: minor question. Is there any reason of removing "one of"?

Reviewer 2: @surbhig12 Thanks for the description. If you fixed it because of the grammar issue, than please keep the original suggestion.

The supported video formats include WMV, ASF, MP4, 3GP, AVI, MKV, and OGG. The available formats depend on the target device.

> In the case of a video interruption, the state will be one of IDLE, READY or PAUSED due to the resource restriction in the system. You must check the state of the current instance and take appropriate action.

Reviewer: Suggest rephrasing, please verify:

```suggestion
In case there is a video interruption, the state will be either IDLE, READY, or PAUSED because of the resource restriction in the system. You must check the state of the current instance and take appropriate action.
```

The following figure illustrates what happens when the player gets interrupted by the system.

**Figure: Player states when interrupted by system**

> ![Player states when interrupted by system](./media/using_players_interrupted_state.png)

Reviewer: Suggestion: Request you to please remove line 72 (figure caption) as it is repetitive with line 70

     }
     ```

>      The playback is interrupted and the callback is triggered when, for example, some other multimedia application starts. The player state automatically changes to `PLAYER_STATE_PAUSED` or `PLAYER_STATE_IDLE` based on the resource management policy. To get the current state, use the `player_get_state()` function.

Reviewer 2: I think the last part of [the @surbhig12's suggestion](https://github.com/Samsung/tizen-docs/pull/1059#discussion_r398315092) is missing.
Would you please check?

```suggestion
     The playback is interrupted and the callback is triggered when, for example, some other multimedia application starts. The player state automatically changes to `PLAYER_STATE_PAUSED` or `PLAYER_STATE_IDLE` based on the resource management policy. To get the current state, use `player_get_state()`.
```

Author: I pushed new commit. thank you.

     }
     ```

>      The playback is interrupted and the callback triggered when, for example, another multimedia application is run. The player state automatically changes to `PLAYER_STATE_PAUSED` or `PLAYER_STATE_IDLE` according to the resource management policy. To get the current state, use the `player_get_state()` function.

Reviewer: Suggest rephrasing, please verify whether my understanding is correct:

```suggestion
The playback is interrupted and the callback is triggered when, for example, some other multimedia application is running. The player state automatically changes to `PLAYER_STATE_PAUSED` or `PLAYER_STATE_IDLE` based on the resource management policy. To get the current state, use `player_get_state()`.
```

Reviewer 2: **Original :** ... for example, another multimedia application is run.
**Suggested :** ... for example, some other multimedia application is running.

I think @eunhaechoi should check the meaning of each sentences as @surbhig12 asked.
To me, the original sentence seems to mean the other application gets started.
However the suggested sentence sounds like just checking the running state of the application.

So, would you please check if my understanding is correct?

Author: How about using " is run " as original expression? If player acquired H/W resources and it is not enough to share with other player, it will be interrupted regardless of player state. The resource is acquired to be PAUSED state and keep in PAUSED and PLAYING state.

Reviewer 2: I also think keeping the original is better. @surbhig12 Would you please let us know your opinion?

Reviewer 2: @eunhaechoi I think @surbhig12's suggestion is good. What's your opinion?

Author: @mijin85cho @surbhig12 I agree. thank you for your suggestion.
